### ISSUE TITLE
[4.8.0] Sparse Prime Tower base

### ISSUE THREAD

**Issue by Block137** [Ultimaker/Cura#9617]
### Application Version

4.8.0

### Platform

Windows 10

### Printer

Custom FFF

### Reproduction steps

Slice this project [project.zip](https://github.com/Ultimaker/Cura/files/6344255/project.zip)
The actual cause is still unknown
Edit: The problem disappeared after switching material profiles around so that the outer wall of the prime tower is NOT printed with Extruder2(T1). I knew that the wall prime tower wall order depends on "Adhesion Tendency" which is inaccessible. Assigning "Generic PLA" to an extruder1(T0) moved it to outer most wall which fixed the problem.

### Actual results

Base of the Prime Tower is sparse, see attached image:
![Screenshot (164)](https://user-images.githubusercontent.com/23495338/115413713-99072400-a21f-11eb-9082-49e35e4796dd.png)

### Expected results

Prime Tower base completely filled for better adhesion.
![Screenshot (165)](https://user-images.githubusercontent.com/23495338/115413734-9e646e80-a21f-11eb-95c1-e178170ad31b.png)

**Comment by fvrmr** [comment]
Hi @Block137 thank you for your bug report. 
I will discuss this with the team. Keep you posted!

**Comment by Ghostkeeper** [comment]
Investigating what happened to this ticket, it got discussed with some people and it was said that this was thought to be a duplicate but the original needed to be found. However that never happened.

I tried reproducing the issue and found the culprit. It was mixing up a list of paths for the prime tower for each extruder, mismatching the paths with the wrong extruders. I've fixed that up and it should work correctly then for version 4.10 and later.
